jax/lax/lax.py

```diff
@@ -369,9 +370,21 @@ def concatenate(operands, dimension):
   return concatenate_p.bind(*operands, dimension=dimension,
                             operand_shapes=tuple(o.shape for o in operands))
 
+# TODO(phawkins): remove once the minimum Jaxlib version is increased to 0.1.22.
+_supports_precision = hasattr(xla_client, "PrecisionConfig")
+
+if _supports_precision:
+  Precision = xla_client.PrecisionConfig.Precision
+else:
+  # Dummy for backward compatibility with older Jaxlib versions.
+  class Precision(enum.Enum):
+    DEFAULT = 0
+    HIGH = 1
+    HIGHEST = 2
+
 def conv_general_dilated(lhs, rhs, window_strides, padding, lhs_dilation=None,
                          rhs_dilation=None, dimension_numbers=None,
-                         feature_group_count=1):
+                         feature_group_count=1, precision=None):
```

Comment:
Instead of None, should we just have a default precision enum value? Though maybe this is for backward compatibility.

In any case, I am just curious :)

Reply:
This is partially for backward compatibility. We could make our own enum and add another option to it (note `DEFAULT` means XLA's default, which is something different to "JAX can choose, e.g., from the trace context").

I'm tempted to leave it as is for now, since we will certainly have to accept `None` here in the future no matter what.

Reply:
Indeed, leave it as it is, thanks for explaining!